mautic/mautic
# Bug 12241 - check list exists before looping

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [yes ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12241 



#### Description:

The key `list` in the `group` Twig template can be looped without checking if it actually exists. So wrapped that loop in an `if`-statement.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. See #12241
